homeassistant/components/magichome/switch.py

```diff
@@ -0,0 +1,41 @@
+"""Support for MagicHome switches."""
```

Comment:
Please limit initial contributions to a single platform (light or switch) and keep other platforms for subsequent PRs.

Reply:
Sorry! I don't quite understand what this means.

Reply:
Hi @balloob! 
May I ask how to limit single platform?
Such as define a PLATFORM_SCHEMA instead of a CONFIG_SCHEMA??

Reply:
Hello @balloob 
is there only one setup_platform for light and switch?